consensus/polybft/consensus_runtime.go

```diff
@@ -167,6 +173,13 @@ func newConsensusRuntime(log hcf.Logger, config *runtimeConfig) (*consensusRunti
 		return nil, fmt.Errorf("consensus runtime creation - restart epoch failed: %w", err)
 	}
 
+	tracker, err := slashing.NewDoubleSigningTracker(logger.Named("double_sign_tracker"), config.State.StakeStore)
```

Comment:
Can we move this to a separate function to be consistent to other init functions like `initGovernanceManager`, `initStakeManager`, etc.

Reply:
Fixed https://github.com/0xPolygon/polygon-edge/pull/1808/commits/437e00809243655bfaa6e1d26859197d6c407c42